frame/bags-list/src/tests.rs

```diff
@@ -196,6 +196,216 @@ mod pallet {
 			assert_storage_noop!(assert!(BagsList::rebag(Origin::signed(0), 10).is_ok()));
 		})
 	}
+
+	#[test]
+	fn put_in_front_of_with_two_node_bag() {
```

Comment:
if these tests are logically the same as the one as before, except they are called through `Dispatchable`, then they have no added value and should be removed. 

A good way to think about it is that the origin of this call is irrelevant, it can be called by any account as long as it signed and pays. So testing `List::put_in_front_of` is enough.

Reply:
IMO if we want to remove one set of these redundant tests we should remove those in the `list` module and leave the ones on the extrinsic. The idea being we test the public api so we can catch any _exact_ regression to the user's interface

Reply:
Remove the relevant `List` module tests here 60743ccfce9cb8a85d8a817e009d4a3bb21ff771